xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -99,12 +99,18 @@ bool CPVRChannelGroup::Load(
   Unload();
 
   int iChannelCount = m_iGroupId > 0 ? LoadFromDb() : 0;
-  CLog::LogFC(LOGDEBUG, LOGPVR, "{} channels loaded from the database for group '{}'",
-              iChannelCount, GroupName());
+  CLog::LogFC(LOGDEBUG, LOGPVR, "Fetched {} {} group members from the database for group '{}'",
+              iChannelCount, IsRadio() ? "radio" : "TV", GroupName());
 
   for (const auto& groupMember : m_members)
   {
     auto channel = channels.find(groupMember.first);
+    if (channel == channels.end())
+    {
+      CLog::Log(LOGERROR, "Cannot find group member '{},{}' in channels!", groupMember.first.first,
+                groupMember.first.second);
+      // No workaround here, please. We need to find and fix the root cause of this case!
+    }
```

Comment:
With a note, I don't really understand here, is the following crash accepted and only the previous log entry as information from where?

The following `(*channel).second` would probably not work :thinking: .

Reply:
Or has you forgot a `return false;`?

Reply:
Maybe on the log, add also the amount of channels, think good to know:
```cpp
      CLog::Log(LOGERROR, "Cannot find group member '{},{}' in {} channels!", groupMember.first.first,
                groupMember.first.second, channels.size());
```

Reply:
inconsistency of the TV database occurred twice with a distance of about 2 weeks at my end. both times after a clean shutdown of kodi. we didn‘t really find the root cause, but this pr corrects the db then and throws out the warning…

doesnt mean it can never happen again but it can cope with it now

Reply:
The comment means that I want to crash (due to the following call to `(*channel).second`, yes. I also could assert() or throw. I forgot nothing and was hoping the comment makes the intention clear.